README.md

```diff
@@ -91,6 +91,15 @@ export __bp_enable_subshells="true"
 ```
 This is disabled by default due to buggy situations related to to `functrace` and Bash's `DEBUG trap`. See [Issue #25](https://github.com/rcaloras/bash-preexec/issues/25)
 
+## Library authors
+If you want to detect bash-preexec in your library (for example, to add hooks to `preexec_functions` when available), use the Bash variable `${bp_imported}`:
```

Comment:
Don't think the `$` and brackets are necessary here. I'd just have this say `bp_imported`.